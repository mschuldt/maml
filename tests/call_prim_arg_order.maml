arduino = Arduino(desktop=True)

@block
def test():
    test_print_args2(test_print(1),test_print(2))
    test_print_args3(test_print(1),test_print(2),test_print(3))
    test_print_args4(test_print(1),test_print(2),test_print(3),test_print(4))
    test_print_args5(test_print(1),test_print(2),test_print(3),test_print(4),test_print(5))
    test_print_args6(test_print(1),test_print(2),test_print(3),test_print(4),test_print(5),test_print(6))
    die(0)
    
arduino.send(test)
