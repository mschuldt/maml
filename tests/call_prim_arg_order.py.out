Initializing avm...
Ready.

ERROR: unrecognized bytecode: '24'
-> 1
-> 2
args=> 1, 2
-> 1
-> 2
-> 3
args=> 1, 2, 3
-> 1
-> 2
-> 3
-> 4
args=> 1, 2, 3, 4
-> 1
-> 2
-> 3
-> 4
-> 5
args=> 1, 2, 3, 4, 5
-> 1
-> 2
-> 3
-> 4
-> 5
-> 6
args=> 1, 2, 3, 4, 5, 6
bye.
